server-api/routes/products.js

```diff
@@ -0,0 +1,99 @@
+const express = require('express');
+const product = require('../models/product');
+const router = express.Router();
+
+//Getting All
+router.get('/', async (req, res) => {
+    try {
+        const products = await product.find();
+        res.json(products);
+    } catch (err) {
+        res.status(500).json({message: err.message})
+    }
+});
+
+//Getting One using ID
```

Comment:
same comment as above - update all comments in this file to follow the correct format: `// this is a comment`